frontend/src/styles/index.styles.ts

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components';
+
+export const MainPage = styled.div`
+    background-color: #edece5;
+    padding: 10px 70px 0px 70px;
+    font-family:  -apple-system, Roboto, Oxygen,
+      Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    
+`;
+export const BodyIndex = styled.div`
+    height:auto;
+    margin-top: 80px;
+    display: flex;
+    justify-content: space-between;
+`;
+export const LeftContainerIndex = styled.div`
+    width: 50%;
+`; 
+export const RightContainerIndex = styled.div`
+    width: 50%;
+`; 
+export const ContainerImg = styled.div`
+    height: 400px;
+    background-color: #e2d8ce;
+    border-radius: 15px;
+    display: flex;
+    justify-content: center;
+    img{
+        height: 400px;
+        padding-bottom: 17px;
+    }
+`;
+export const Header = styled.div`
+    height: auto;
+    border-bottom: solid 2px #f5f5f1;
+    display: flex;
+    justify-content: space-between;
+    img{
+        height: 90px;
+        margin-left: -32px;
+    }
+`;
+export const ButtonLoginHeader = styled.div`
+    background-color: #33b6a6;
+    border-radius: 9px;
+    padding: 10px 20px 10px 20px;
```

Comment:
```suggestion
    padding: 10px 20px;
```

Better to manutention.